### ISSUE TITLE
pca.Rmd

### ISSUE THREAD

**Issue by topepo** [marlycormar/applicability_domain#1]
Looks good. A few notes on the first version:

1. When you load `AmesHousing`, you can just use `ames <- make_ames()` to get an analysis ready version of the data set. 

1. You need a `library(shiny)` declaration at the start to use `inputPanel()`

1. There's an `rsample` function called `initial_split()` that is a more `tidymodels` method for making the training and test sets. In our training notes, we've used:

```r
library(AmesHousing)
ames <- 
  make_ames() %>% 
  # Remove quality-related predictors
  dplyr::select(-matches("Qu"))

set.seed(4595)
data_split <- initial_split(ames, strata = "Sale_Price")

ames_train <- training(data_split)
ames_test  <- testing(data_split)
``` 

4. In addition to looking at the PCA scores for the training set, we'd be more interested in looking at the test set scores to see if they are "like" the distribution of their training set counterparts.

**Comment by marlycormar** [comment]
Hi @topepo, thanks for your helpful feedback!! I went over your comments and addressed them in PR #2.

**Comment by marlycormar** [comment]
Addressed by PR #2.
